Compile Error

When compile this module i get this error:

```
'/usr/include/glib-2.0/gio/gcredentials.h:72:1: error: declaration of 'pid_t' must be imported from module 'SwiftGlibc.POSIX.termios' before it is required
pid_t            g_credentials_get_unix_pid       (GCredentials    *credentials,'

'/usr/include/termios.h:30:17: note: previous declaration is here
typedef __pid_t pid_t;'
```

--- a/headers.h
+++ b/headers.h
@@ -1,0 +1,1 @@
+#include <termios.h>

--- a/module.modulemap
+++ b/module.modulemap
@@ -1,4 +1,5 @@
 module CGtk [system] {
+  header "headers.h"
   header "/usr/include/gtk-3.0/gtk/gtk.h"
 
   link "gtk-3"

